src/main/kotlin/tech/thatgravyboat/skycubed/features/map/screen/MapsWidget.kt

```diff
@@ -99,9 +99,10 @@ class MapsWidget(
                     }
 
                     if (map.island == LocationAPI.island) {
-                        Waypoints.waypoints().forEach { waypoint ->
-                            val mapX = waypoint.pos.x - 3 + map.offsetX + width / 2f
-                            val mapY = waypoint.pos.z - 3 + map.offsetY + height / 2f
+                        MeowddingWaypointHandler.waypoints.forEach { waypoint ->
```

Comment:
id like it to just be skycubed created ones

Reply:
its more about being able to delete another mods waypoint, like for example deleting the metal detector waypoint